src/submission/exceptions/exceptions.ts

```diff
@@ -161,52 +189,56 @@ export const checkForProgramAndEntityExceptions = async ({
 
   // if there are submitted exceptions for this program, check if they match record values
   if (!(programException || entityException)) {
-    return { filteredErrors: schemaValidationErrors, normalizedRecord: record };
+    return { filteredErrors: schemaValidationErrors, normalizedRecord };
   }
 
   // check each validation error for a matching exception, and remove the validaiton if the value matches the exception value
   schemaValidationErrors.forEach(validationError => {
     const validationErrorFieldName = validationError.fieldName;
     const fieldValue = record[validationErrorFieldName];
 
-    // Field value is a type we cannot allow exceptions for, validate as normal
-    if (!isAllowedTypeForException(fieldValue)) {
-      filteredErrors.push(validationError);
-      return;
-    }
+    const fieldSchema = entitySchema?.fields.find(fieldFilter(validationErrorFieldName));
+    const valueType = fieldSchema?.valueType;
+
+    const validNumericExceptionValue =
+      (valueType === 'number' || valueType === 'integer') &&
```

Comment:
Can reuse the `isNumericType` function I suggested in the other comment.

Reply:
Updated